pkg/api/serialization_test.go

```diff
@@ -187,6 +187,11 @@ func fuzzInternalObject(t *testing.T, forVersion unversioned.GroupVersion, item
 			j.DockerImageMetadataVersion = []string{"pre012", "1.0"}[c.Rand.Intn(2)]
 			j.DockerImageReference = c.RandString()
 		},
+		func(j *image.ImageSignature, c fuzz.Continue) {
```

Comment:
why does this need fields excluded from fuzzing?  You should also do it, `c.FuzzNoCustom` and then clear the fields you can't set.